src/io/flutter/run/bazel/BazelFields.java

```diff
@@ -127,8 +127,12 @@ BazelFields copy() {
 
   @NotNull
   BazelFields copyTemplateToNonTemplate(@NotNull final Project project) {
-    final Workspace workspace = WorkspaceCache.getInstance(project).getNow();
+    final WorkspaceCache workspaceCache = WorkspaceCache.getInstance(project);
```

Comment:
Also, I wonder if we should add a helper method to simplify the callers:

WorkspaceCache.getWorkspaceNow()